src/Diagnostics.RuntimeHost/Models/PostBodyModel.cs

```diff
@@ -73,4 +73,14 @@ public class BodyValidationResult
         public bool Status { get; set; }
         public string Message { get; set; }
     }
+
+    public class DiagnosticWorkerAppData
+    {
+        public string WorkerAppName { get; set; }
+        public string Tags { get; set; }
+        public string ResourceGroupName { get; set; }
+        public string SubscriptionName { get; set; }
+        public string KubeEnvironmentName { get; set; }
+        public string GeoMasterInstance { get; set; }
```

Comment:
Just a reminder that we just updated the response structure returned by Observer to
![image](https://user-images.githubusercontent.com/68874830/132397863-d5e1c85a-9c2f-4ff8-8355-846c5f903d6a.png)

Reply:
Yes, I will be incorporating these changes soon.